docker/pytorch/Dockerfile

```diff
@@ -149,3 +149,34 @@ RUN useradd -rm -d /home/mosaicml -s /bin/bash -u 1000 -U -s /bin/bash mosaicml
 WORKDIR /home/mosaicml
 
 USER mosaicml
+
+
+FROM pytorch_stage AS vision_stage
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG PYTORCH_VERSION
```

Comment:
Nit: Do args used in a previous stage need to be declared again? (Not sure, but if not, can we remove?)